plugin/trino-opa/src/main/java/io/trino/plugin/opa/OpaHighLevelClient.java

```diff
@@ -190,4 +204,18 @@ private static OpaQueryInput buildQueryInputForSimpleAction(OpaQueryContext cont
     {
         return new OpaQueryInput(context, OpaQueryInputAction.builder().operation(operation).build());
     }
+
+    private static Map<String, String> loadAdditionalContextFromFile(Optional<File> additionalContextFile)
+    {
+        if (additionalContextFile.isEmpty()) {
+            return Map.of();
+        }
+
+        try {
+            return OBJECT_MAPPER.readValue(additionalContextFile.get(), new TypeReference<Map<String, String>>() {});
```

Comment:
I don't see a point of having a JSON file if we read it to Map<String, String>

ordinary *.properties file would do.